meson_options.txt

```
@@ -14,10 +14,8 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with crest.  If not, see <https://www.gnu.org/licenses/>.
 
-option(
-  'la_backend',
-  type: 'combo',
-  value: 'mkl-static',
-  choices: ['mkl', 'mkl-static', 'openblas', 'netlib'],
-  description: 'Linear algebra backend for program.',
-)
+option('la_backend', type: 'combo', value: 'mkl-static',
+       choices: ['mkl', 'mkl-rt', 'mkl-static', 'openblas', 'netlib', 'custom'],
+       description: 'Linear algebra backend for program.')
+option('custom_libraries', type: 'array', value: [],
+       description: 'libraries to load for custom linear algebra backend')
```

Let's use a git diff friendly format here
```suggestion
option(
  'la_backend',
  type: 'combo',
  value: 'mkl-static',
  choices: ['mkl', 'mkl-rt', 'mkl-static', 'openblas', 'netlib', 'custom'],
  description: 'Linear algebra backend for program.',
)
option(
  'custom_libraries',
  type: 'array',
  value: [],
  description: 'libraries to load for custom linear algebra backend',
)
```